<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AptAnalyzerWindow</class>
 <widget class="QWidget" name="AptAnalyzerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1355</width>
    <height>903</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="title_layout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>关键词：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="e_search"/>
        </item>
        <item>
         <widget class="QPushButton" name="btn_search">
          <property name="text">
           <string>查询</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QFrame" name="central">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QSplitter" name="splitter">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QGroupBox" name="groupBox">
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="title">
             <string>发行版与源</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <property name="leftMargin">
              <number>4</number>
             </property>
             <property name="topMargin">
              <number>4</number>
             </property>
             <property name="rightMargin">
              <number>4</number>
             </property>
             <property name="bottomMargin">
              <number>4</number>
             </property>
             <item>
              <widget class="QListWidget" name="aptDistroList">
               <item>
                <property name="text">
                 <string>debian/bookworm</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QTabWidget" name="tabWidget">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="tab">
             <attribute name="title">
              <string>概览</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_17">
                <item>
                 <widget class="QLabel" name="label_17">
                  <property name="text">
                   <string>源地址：</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="r_Url">
                  <property name="text">
                   <string>Url</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_17">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>源名称：</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="r_Origin">
                  <property name="text">
                   <string>Origin</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>源标识：</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="r_Label">
                  <property name="text">
                   <string>Label</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_4">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>源套件：</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="r_Suite">
                  <property name="text">
                   <string>Suite</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_5">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>源版本：</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="r_Version">
                  <property name="text">
                   <string>Version</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_6">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>源代号：</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="r_Codename">
                  <property name="text">
                   <string>Codename</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_7">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_8">
                <item>
                 <widget class="QLabel" name="label_7">
                  <property name="text">
                   <string>源变化：</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="r_Changelogs">
                  <property name="text">
                   <string>Changelogs</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_8">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_9">
                <item>
                 <widget class="QLabel" name="label_8">
                  <property name="text">
                   <string>源日期：</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="r_Date">
                  <property name="text">
                   <string>Date</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_9">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="0,0,0">
                <item>
                 <widget class="QLabel" name="label_9">
                  <property name="text">
                   <string>源日期：</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="r_Acquire">
                  <property name="text">
                   <string>Acquire</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_10">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_11">
                <item>
                 <widget class="QLabel" name="label_10">
                  <property name="text">
                   <string>源架构：</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="r_Architectures">
                  <property name="text">
                   <string>Architectures</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_11">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_12">
                <item>
                 <widget class="QLabel" name="label_11">
                  <property name="text">
                   <string>源组成：</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="r_Components">
                  <property name="text">
                   <string>Components</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_12">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_13">
                <item>
                 <widget class="QLabel" name="label_12">
                  <property name="text">
                   <string>源简介：</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="r_Description">
                  <property name="text">
                   <string>Description</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_13">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>441</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_2">
             <attribute name="title">
              <string>源与包列表</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_16">
                <item>
                 <widget class="QLabel" name="label_15">
                  <property name="text">
                   <string>架构成分：</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="lab_Architectures">
                  <property name="text">
                   <string>lab_Architectures</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_15">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_15">
                <item>
                 <widget class="QLabel" name="label_16">
                  <property name="text">
                   <string>组件成分：</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="lab_Components">
                  <property name="text">
                   <string>lab_Components</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_16">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_14">
                <item>
                 <widget class="QLabel" name="label_13">
                  <property name="text">
                   <string>选择架构：</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comb_Architectures">
                  <property name="minimumSize">
                   <size>
                    <width>100</width>
                    <height>0</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="Line" name="line">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_14">
                  <property name="text">
                   <string>选择组件：</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comb_Components">
                  <property name="minimumSize">
                   <size>
                    <width>100</width>
                    <height>0</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_14">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QProgressBar" name="apt_loading_progress">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>10</height>
                 </size>
                </property>
                <property name="value">
                 <number>24</number>
                </property>
                <property name="textVisible">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTabWidget" name="tabWidget_Packages">
                <property name="currentIndex">
                 <number>1</number>
                </property>
                <widget class="QWidget" name="tab_3">
                 <attribute name="title">
                  <string>数据</string>
                 </attribute>
                 <layout class="QGridLayout" name="gridLayout_2">
                  <item row="0" column="0">
                   <widget class="QTextBrowser" name="apt_packages_browser"/>
                  </item>
                 </layout>
                </widget>
                <widget class="QWidget" name="tab_4">
                 <attribute name="title">
                  <string>表结果</string>
                 </attribute>
                 <layout class="QGridLayout" name="gridLayout_3">
                  <item row="0" column="0">
                   <widget class="QStackedWidget" name="stacked_apt_packages">
                    <widget class="QWidget" name="stack_table">
                     <layout class="QGridLayout" name="gridLayout">
                      <property name="leftMargin">
                       <number>0</number>
                      </property>
                      <property name="topMargin">
                       <number>0</number>
                      </property>
                      <property name="rightMargin">
                       <number>0</number>
                      </property>
                      <property name="bottomMargin">
                       <number>0</number>
                      </property>
                      <item row="0" column="0">
                       <widget class="QTableWidget" name="apt_packages_table">
                        <column>
                         <property name="text">
                          <string>包名</string>
                         </property>
                        </column>
                        <column>
                         <property name="text">
                          <string>版本</string>
                         </property>
                        </column>
                        <column>
                         <property name="text">
                          <string>下载地址</string>
                         </property>
                        </column>
                       </widget>
                      </item>
                     </layout>
                    </widget>
                    <widget class="QWidget" name="stack_loading">
                     <layout class="QGridLayout" name="gridLayout_4">
                      <property name="leftMargin">
                       <number>0</number>
                      </property>
                      <property name="topMargin">
                       <number>0</number>
                      </property>
                      <property name="rightMargin">
                       <number>0</number>
                      </property>
                      <property name="bottomMargin">
                       <number>0</number>
                      </property>
                      <item row="0" column="0">
                       <widget class="QLabel" name="label_18">
                        <property name="text">
                         <string>正在加载中...</string>
                        </property>
                        <property name="alignment">
                         <set>Qt::AlignCenter</set>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </widget>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
